RecoBTag/Combined/python/pfDeepCSVJetTags_cfi.py

```diff
@@ -12,13 +12,17 @@
 	)
 
 from Configuration.Eras.Modifier_phase1Pixel_cff import phase1Pixel
-phase1Pixel.toModify(pfDeepCSVJetTags, NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'))
-phase1Pixel.toModify(pfDeepCSVJetTags, checkSVForDefaults = cms.bool(True))
-phase1Pixel.toModify(pfDeepCSVJetTags, meanPadding = cms.bool(True))
-phase1Pixel.toModify(pfDeepCSVJetTags, toAdd = cms.PSet())
+phase1Pixel.toModify(pfDeepCSVJetTags, 
+                     NNConfig = cms.FileInPath('RecoBTag/Combined/data/DeepCSV_PhaseI.json'),
```

Comment:
if `NNConfig` already exists, you can drop also `cms.FileInPath`

Reply:
> if `NNConfig` already exists, you can drop also `cms.FileInPath`

Indeed, thank you @silviodonato 
The same at L24 below
We'll submit a quick patch as soon as this PR gets included in a IB

Reply:
Finalization in #31568 (submitted now before forgetting it)